picard_mark_duplicates_4.1.8.1/README.md

```diff
@@ -0,0 +1,77 @@
+# CWL for running Picard - MarkDuplicates
+
+## Version of tools in docker image
+
+| Tool	| Version	| Location	|
+|---	|---	|---	|
+| picard  	| 2.21.2  	|  https://github.com/broadinstitute/picard/releases/download/2.21.2/picard.jar	|
```

Comment:
I'd suggest updating the readme to reflect that you are using gatk version of picard tools